### ISSUE TITLE
Multitouch

### ISSUE THREAD

**Issue by NicolasParada** [SuneBear/Piano-Flow#1]
Multitouch is not working. I have to quit my finger from touching the screen to be able to touch again.

Device: Windows Laptop, HP Spectre x360.

**Comment by SuneBear** [comment]
Uh-oh, I don't have a touchable Windows Laptop. I'm not sure touch event is available when your browser user-agent is PC environment. Can you provide more details about the question?

**Comment by SuneBear** [comment]
#2

**Comment by NicolasParada** [comment]
Sorry, I tried to look up in the code, but it's pretty similar to touchpianist. I could find what was the difference and why touchpianist works with multiple touch points and this don't.

**Comment by SuneBear** [comment]
Yeah, you are good at observation. The game core including performer, visual pitch and audio synthesis was taken from Touch Pianist in the current version v0.0.x for learning MUG. I will re-write a new one with a new MVVM architecture, and some different game modes. I wonder why multitouch in Piano Flow does not work on Windows?
